File: SmartHR/SmartHR.DataApi/ViewModels/Data/WorkHourViewModel.cs
using SmartHR.DataApi.Models.Constants;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SmartHR.DataApi.ViewModels.Data
{
    public class WorkHourViewModel
    {
        public int WorkHourId { get; set; }
        public OfficeHourType OfficeHourType { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime LeaveTime { get; set; }

        public DateTime BreakTime { get; set; }

        public int BreakDuration { get; set; }
    }
}

File: SmartHR.Client/ClientApp/src/app/models/data/viewmodel/work-hour-view-model.ts
import { OfficeHourType } from "../constants/enum-data";

export class WorkHourViewModel {
  constructor(
    public workHourId?: number,
    public officeHourType?: OfficeHourType,
    public startTime?: Date | string,
    public leaveTime?: Date | string,
    public breakTime?: Date | string,
    public breakDuration?: number
  ) { }
}
